.mocharc.yml

```diff
@@ -0,0 +1 @@
+require: 'ts-node/register/transpile-only'
```

Comment:
Reviewer note: ts-node offers this way to specify the `transpileOnly: true` setting (which is what ts-mocha was using)

Reply:
nice finding 🎉